commit 2d5f15c865de75f3147bd849ef81d3ed034f8104
Author: Darrell Anderson <humanreadable@yahoo.com>
Date:   1335654338 -0500

    Reverse patch from GIT hash 33e649c9.
    This resolves bug report 406.
    Thanks to Slavek Banko.

diff --git a/kmail/kmcomposewin.cpp b/kmail/kmcomposewin.cpp
index 227f48c..cb9db70 100644
--- a/kmail/kmcomposewin.cpp
+++ b/kmail/kmcomposewin.cpp
@@ -4351,8 +4351,6 @@ void KMComposeWin::slotContinueDoSend( bool sentOk )
       }
       (*it)->cleanupHeader();
       sentOk = kmkernel->msgSender()->send((*it), mSendMethod);
-      if (sentOk)
-          kmkernel->acctMgr()->checkMail(true);
     }
 
     if (!sentOk)
